package/gluon-core/luasrc/lib/gluon/upgrade/115-swconfig

```diff
@@ -5,6 +5,7 @@ local uci = require('simple-uci').cursor()
 
 local switch_vlans = {
 	-- device identifier, lan ports, wan ports
+	["ubnt,unifiac-mesh-pro"] = {"2 0t", "3 0t"},
```

Comment:
I think these should be swapped.

Reply:
resolved in 18d52679d68b14beb12a924c189d28912ac746a4